### PR TITLE
test: add more RDNSS testing for PR#193

### PR DESCRIPTION
**Pull request by robbat2**
Test harnesses ready for PR193
Should pass right now, but adding more RDNSS servers should trigger failure.
- more than 3 on a single block -> parsing failure
- multiple blocks -> success with multiple options

Reference: https://github.com/radvd-project/radvd/pull/193
Signed-off-by: Robin H. Johnson <robbat2@gentoo.org>

### CODE REVIEW

**Comment by the-j0k3r** [comment]
We had merged this patch downstream, I did not test it myself but clearly it caused some instability and was reverted

One such report https://forum.dd-wrt.com/phpBB2/viewtopic.php?p=1275031#1275031

>Same thing happened to me. If you clean those two ipv6 dns lines (leave them empty) radvd deamon will not crash. Your/my /tmp/radvd.conf look/ed like:

>root@tp:/tmp# cat radvd.conf
>interface br0
>{
>IgnoreIfMissing on;
>AdvSendAdvert on;
>MinRtrAdvInterval 3;
>MaxRtrAdvInterval 10;
>AdvHomeAgentFlag off;
>AdvManagedFlag off;
>AdvOtherConfigFlag on;
>AdvLinkMTU 1452;
>prefix ::/64
>{
>AdvOnLink on;
>AdvAutonomous on;
>AdvValidLifetime 30;
>AdvPreferredLifetime 20;
>};
>RDNSS 2606:4700:4700::1111 2606:4700:4700::1001{};
>};

>On 50786 once you leave those boxes empty you'll end up whith the same conf sans that RDNSS line and radvd won´t die.

>Syslog entries of radvd deamon dying:

>Nov 3 07:07:12 tp daemon.info radvd[9224]: version 2.19 started
>Nov 3 07:07:12 tp daemon.err radvd[9226]: Exiting, privsep_read_loop had readn return 0 bytes
>Nov 3 07:07:12 tp daemon.err radvd[9226]: Exiting, privsep_read_loop is complete.

>Previous firm with that line working: 50485.

File: test/test1.conf [hunk 1]
```diff
@@ -71,7 +74,7 @@ interface eth0 {
 	abro fe80::a200:0:0:1/64 {
 		AdvVersionLow 10;
 		AdvVersionHigh 2;
-		AdvValidLifetime 2;
+		AdvValidLifeTime 2;
```
Comment (stappersg) [inline test/test1.conf:77]: Changing the `...Lifetime` into `...LifeTime` (time with T) feels wrong when seeing `AdvRDNSSLifetime` (time with t).

For the record: I will approve this merge request so the radvd project  can move on.

**Review by stappersg — APPROVED** [review]
Approved for making moving forward possible.

In other words: Approved for circumventing the blocking _"no self approval"_ setting that github enforces.

**Comment by stappersg** [comment]
image

If this _merge request_ now waits on me, please say so.

**Pull request merged**